tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -124,10 +140,10 @@ def test_clone_create_delete_performance(self, interface_type, pvc_size, file_si
             clone_yaml = constants.CSI_CEPHFS_PVC_CLONE_YAML
         file_size_mb = convert_device_size(file_size, "MB")
 
-        # creating single clone ( or many one by one if max_mum_of_clones > 1)
-        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size} GB.")
+        # creating clone(s)
+        # logger.info(f"Start creating {clones_num} clones on {interface_type} PVC of size {pvc_size} GB.")
 
-        for i in range(max_num_of_clones):
+        for i in range(clones_num):
             logger.info(f'Start creation of clone number {i + 1}.')
             cloned_pvc_obj = pvc.create_pvc_clone(sc_name, parent_pvc, clone_yaml, storage_size=pvc_size + "Gi")
```

Comment:
creating objects for each clone is memory intensive like we've seen in the past. we may need to enhance this later.